examples/resources/awscc_cassandra_table/table_tags.tf

```
@@ -0,0 +1,35 @@
+resource "awscc_cassandra_keyspace" "awscc_cassandra_keyspace_example" {
+  keyspace_name = "awscc_cassandra_keyspace_example"
+}
+
+resource "awscc_cassandra_table" "awscc_cassandra_table_example" {
+  keyspace_name = "awscc_cassandra_keyspace_example"
+  table_name    = "awscc_cassandra_table_example"
+  partition_key_columns = [{
+    column_name = "Message"
+    column_type = "ASCII"
+  }]
+  regular_columns = [{
+    column_name = "name"
+    column_type = "TEXT"
+    }, {
+    column_name = "region"
+    column_type = "TEXT"
+    }, {
+    column_name = "role"
+    column_type = "TEXT"
+    }, {
+    column_name = "vacation_hrs"
+    column_type = "FLOAT"
+    }
+  ]
+  tags = [{
+    "key"   = "Name"
+    "value" = "awscc"
+    },
+    {
+      "key"   = "Componet"
+      "value" = "Cassandra"
+  }]
+  depends_on = [awscc_cassandra_keyspace.awscc_cassandra_keyspace_example]
+}
```

```suggestion
resource "awscc_cassandra_table" "awscc_cassandra_table_example" {
  keyspace_name = awscc_cassandra_keyspace.awscc_cassandra_example.id
  table_name    = "awscc_cassandra_table_example"
  partition_key_columns = [{
    column_name = "Message"
    column_type = "ASCII"
  }]
  regular_columns = [{
    column_name = "name"
    column_type = "TEXT"
    }, {
    column_name = "region"
    column_type = "TEXT"
    }, {
    column_name = "role"
    column_type = "TEXT"
    }, {
    column_name = "vacation_hrs"
    column_type = "FLOAT"
    }
  ]
  tags = [{
    "key"   = "Name"
    "value" = "awscc"
    },
    {
      "key"   = "Component"
      "value" = "Cassandra"
  }]
}
```

Fixed typo on tag key "Components" and remove explicit `depends_on`

---

Changes made in the latest commit 7b66b65b7b778bfbb3e3d77b3ecaa22609c97665